prow/resultstore/files.go

```diff
@@ -34,16 +34,26 @@ type fileFinder interface {
 	Attributes(ctx context.Context, name string) (pio.Attributes, error)
```

Comment:
Commit message still references "MustFiles".

s/allows us add/allows us to add

Reply:
Done